src/components/form/FormContainer.jsx

```diff
@@ -0,0 +1,189 @@
+import React, { useEffect, useState } from 'react';
+import { Heading, Paragraph } from '../shared';
+import Inputs from '../shared/partials/FormInputs';
+import formimg from '../../assets/images/form-tickets.png';
+import { feeCoverage, initialContent, inputContent, lastPartContent } from '../../data/formInformation';
+import { storeFormData } from '../../firebase/registration';
+
+const FormContainer = () => {
+	const [formData, setFormData] = useState({});
+	//const [verified, setVerified] = useState(false);
+	// const { userData } = useContext(AuthContext);
+	const userData = undefined;
+	console.log('formData:', formData);
+
+	const setInputValue = (key, value, isValid) => {
+		if (!key || !isValid) return;
+
+		setFormData(prev => ({
+			...prev,
+			[key]: value,
+		}));
+	};
+
+	useEffect(() => {
+		if (userData) {
+			setFormData({
+				recRollNumber: userData?.rollNumber || '',
+				name: userData?.name || '',
+				email: userData?.email || '',
+				county: userData?.county || '',
+				state: userData?.state || '',
+				city: userData?.city || '',
+				prefix: userData?.prefix || '',
+				mobile: userData?.mobile || '',
+				regType: userData?.regType || '',
+			});
+		}
+	}, [userData]);
+
+	const registerUser = async e => {
+		e.preventDefault();
+		try {
+			const documentId = await storeFormData(formData);
+			console.log('Document ID:', documentId);
+			// Reset form after successful submission
+			setFormData({
+				recRollNumber: '',
+				name: '',
+				email: '',
+				country: '',
+				state: '',
+				city: '',
+				contactNumber: '',
+			});
+		} catch (error) {
+			console.error('Error:', error);
+		}
+		console.log('Form Data:', formData);
+	};
+
+	return (
+		<div
+			className='text-white bg-black flex flex-col md:flex-row justify-between py-6'
+			style={{
+				backgroundImage:
+					'url(https://github.com/Srish-ty/project-udon/blob/feat/form/src/assets/images/Ellipse.png?raw=true)',
+				backgroundRepeat: 'no-repeat',
+				backgroundSize: 'cover',
+			}}>
+			<div id='info' className='w-[90%] md:w-[45%] m-8 p-2'>
+				{initialContent.map((item, index) =>
+					index == 0 ? (
+						<Paragraph variant='body3' key={index} className={item.classes}>
+							{item.text}
+						</Paragraph>
+					) : (
+						<Paragraph variant='body3' className={'text-primary ' + item.classes} key={index}>
+							{item.text}
+						</Paragraph>
+					)
+				)}
+				<div id='imgContainer' className='mx-auto my-8 w-[50vw] md:w-[408px]'>
+					<img src={formimg} height={'100%'} width={'100%'} alt='ticket-price' />
+				</div>
+				<Paragraph variant='body3'>
+					Alumni donating/contributing more than ₹50,000 can pay &quot;Individual Only (without family)&quot; registration
+					fee for their family registration and they do not need to pay &quot;With Family (spouse + kids)&quot; registration
```

Comment:
Store the text in a constant file and use it from there

Reply:
fixed .https://github.com/dscnitrourkela/project-udon/pull/11/commits/d71536031b2f4dcc49ea665471d81743bd42579e